src/mjolnir/restrictionbuilder.cc

```diff
@@ -489,7 +424,7 @@ void build(const std::string& complex_restriction_from_file,
                 res_way_ids.push_back(restriction_to.to());
 
                 // walk in the forward direction (reverse in relation to the restriction)
-                std::deque<GraphId> tmp_ids = GetGraphIds(currentNode, reader, lock, res_way_ids);
+                std::vector<GraphId> tmp_ids = GetGraphIds(currentNode, reader, lock, res_way_ids);
```

Comment:
I just don't see sense in using deque here. It just confused me. But I can easily revert it if there's some profit